compiler/codegen/OMRCodeGenerator.cpp

```diff
@@ -236,6 +237,12 @@ OMR::CodeGenerator::CodeGenerator() :
      _symbolDataTypeMap(self()->comp()->allocator()),
      _lmmdFailed(false)
    {
+#ifdef OMR_RELOCATION_RUNTIME
+   if (self()->comp()->compileRelocatableCode())
+      {
+      _aheadOfTimeCompile=new (self()->trHeapMemory()) TR::AheadOfTimeCompile(NULL,self()->comp());
```

Comment:
Because in this case, according to previous discussion in this PR, the header size map is not planned to be used. The idea Irwin expressed in the previous discussion is to issue calls to acquire the sizes to specific classes based on relocation record types. That will also need discussion, but for now the header size maps are not used in our prototype AOT compiler implementation that exercises this code

Reply:
ok, it's a little bit strange to have something here that isn't being used and may change, but i guess it's not too outlandish.

Reply:
Thank you, this is done to support both of the relocation runtimes: the OpenJ9 and OMR

Reply:
This section of the code is mentioned in #5469, hence marking it resolved